nxviz/plots.py

```diff
@@ -711,6 +724,8 @@ def draw_nodes(self):
                         ha=label_ha,
                         va=label_va,
                         color=self.node_label_color[i],
+                        fontsize=self.fontsize,
+                        family=self.fontfamily
```

Comment:
Same comment as above.

```suggestion
                        fontfamily=self.fontfamily
```